amida-tech/amida-messaging-microservice
# Feature/SER-151 GET thread/:originalMessageId endpoint

#### What's this PR do?
Add a `GET /thread/:originalMessageId` to retrieve all of a user's messages sharing that `originalMessageId`

#### Related JIRA tickets:
[SER-151](https://jira.amida-tech.com/browse/SER-151)

#### How should this be manually tested?
1. Populate a thread by sending a message and replying to that message
1. Call `GET /thread/:originalMessageId` and check that all the messages owned by the caller with that originalMessageId are returned.

#### Any background context you want to provide?

#### Screenshots (if appropriate):
